examples/csv/src/main.rs

```diff
@@ -377,16 +378,24 @@ mod tests {
                     hull: Hull,
                 }
 
-                fn unknown_hull_values(v: Vaisseau) -> Option<string_10> {
+                fn is_other_hull(v: &Vaisseau) -> bool {
```

Comment:
It seems that the behavior of the  stream method `filter_map` is no longer being tested. This is not desirable in the test suite of a programming language implementing `filter_map`.